Formula/das_tool.rb

```diff
@@ -0,0 +1,40 @@
+class DasTool < Formula
+  # cite Sieber_2018: "https://doi.org/10.1038/s41564-018-0171-1"
+  desc "Genomic binning refiner"
+  homepage "https://github.com/cmks/DAS_Tool"
+  url "https://github.com/cmks/DAS_Tool/archive/1.1.1.tar.gz"
+  sha256 "2a55f67b5331251d8fd5adea867cc341363fbf7fa7ed5c3ce9c7679d8039f03a"
+  head "https://github.com/cmks/DAS_Tool.git"
+
+  if OS.mac?
+    depends_on "coreutils"
+  else
+    depends_on "unzip"
+  end
+
+  depends_on "diamond"
+  depends_on "prodigal"
+  depends_on "pullseq"
+  depends_on "r"
+
+  def install
+    mkdir_p buildpath/"lib/R"
+    ENV["R_LIBS_SITE"] = "#{buildpath}/lib/R"
+
+    system "Rscript", "-e", "install.packages(c('ggplot2','doMC','data.table'), repo='http://cran.us.r-project.org');"
```

Comment:
How about using `repo='https://cran.rstudio.com'` ? I don't know how it's like now, but I believe it may be faster than `cran.us.r-project.org`.